sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified.json

```diff
@@ -463,7 +463,7 @@
       "delete": {
         "summary": "Delete the identity of a device from the identity registry of an IoT hub.",
         "description": "Delete the identity of a device from the identity registry of an IoT hub.",
-        "operationId": "RegistryManager_DeleteDevice",
+        "operationId": "Devices_Delete",
```

Comment:
You need to distinguish between Identity CRUD and Twin CRUD. You can't just have "Devices_Delete". It should be "Devices_DeleteIdentity" here and everywhere